### DISCUSSION TITLE
windows 10 makcert -install installs expired certificate

### DISCUSSION THREAD

**Discussion by rpuig** [FiloSottile/mkcert#440] [Q&A]
I have windows 10 and installed mkcert with mkcert -install with powersell. Then when I try to use it following the steps. I found that the certificate was created with 11/11/2009 to 09/11/2019 dates. But today its  04/04/2022. any cues as how to solve this? I have tried on chrome, brave .

**Comment by FiloSottile** [comment]
This is weird and I can't think of a way this would happen except if the clock of the machine was off when mkcert first ran.

Try doing `mkcert -uninstall`, deleting the files at `mkcert -CAROOT`, and doing `mkcert -install` again.
